packages/framework-provider-local/package.json

```diff
@@ -35,7 +35,7 @@
     "clean": "rimraf ./dist tsconfig.tsbuildinfo",
     "prepack": "tsc -b tsconfig.json",
     "test:provider-local": "npm run test",
-    "test": "BOOSTER_ENV=test nyc --extension .ts mocha --forbid-only \"test/**/*.test.ts\""
+    "test": "set BOOSTER_ENV=test & nyc --extension .ts mocha --forbid-only \"test/**/*.test.ts\""
```

Comment:
As above, need to find some way to only apply this in windows (or would this work ok on a mac?)

Reply:
if it doesnt work we can switch to cross-env